RealmJS.podspec

```diff
@@ -0,0 +1,97 @@
+require 'json'
+
+package = JSON.parse(File.read(File.expand_path('package.json', __dir__)))
+
+# Building the podspec from a local .git repository when developing - GitHub otherwise
+git_path_local = File.expand_path('.git', __dir__)
+git_path = File.directory?(git_path_local) ? "file://#{git_path_local}" : 'https://github.com/realm/realm-js.git'
+
+app_path = File.expand_path('../..', __dir__)
+node_modules_path = File.expand_path('node_modules', app_path)
+react_native_module_path = File.expand_path('react-native', node_modules_path)
+if not File.directory?(react_native_module_path)
+  raise "Couldn't find React Native node module (#{react_native_module_path}) - did you install `npm install react-native`?"
+end
+
+Pod::Spec.new do |s|
+  s.name                   = "RealmJS"
+  s.version                = package['version']
+  s.summary                = package['description']
+  s.license                = package['license']
+
+  s.authors                = package['author']
+  s.homepage               = package['homepage']
+  s.platform               = :ios, '9.0'
+
+  s.source                 = { :git => git_path, :tag => "v#{s.version}", :submodules => true }
+
+  s.prepare_command        = <<-CMD
+    # Run `npm ci` to install the dependencies the download script needs, if not installed as a node_module
+    ([ -d '../../node_modules' ] || npm ci) && node ./scripts/download-realm.js ios --sync;
+    # Clean up from any previous builds
+    rm -rf include
+    # Build up the include directory
+    mkdir -p include include/jsc include/impl/apple include/util/apple include/sync/impl/apple
```

Comment:
You can skip `include` - `-p` will create it.

Reply:
Yeah - makes sense :)